shared/django_apps/reports/models.py

```diff
@@ -404,7 +404,7 @@ class PartitioningMeta:
     )
     date = models.DateField()
     repoid = models.IntegerField()
-    branch = models.TextField()
+    branch = models.TextField(null=True)
```

Comment:
@joseph-sentry I'll summon thee to see if it'd affect your work

Reply:
ah this is fine i think, it's probably crashing the test results processor at the moment, I find it weird that we have commits with no branch but i guess that's a separate issue